src/App.jsx

```diff
@@ -14,45 +19,88 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const getSortedGoods = (goods, { sortField, isReversed }) => {
+  const sortedGoods = [...goods];
+
+  if (sortField) {
+    sortedGoods.sort((item1, item2) => {
+      switch (sortField) {
+        case 'name':
```

Comment:
You can use created variables here and line 30 (SORT_FIELD_NAME, SORT_FIELD_LENGTH).